scripts/linters/python_linter_test.py

```diff
@@ -40,7 +40,11 @@
     LINTER_TESTS_DIR, 'invalid_docstring.py')
 
 NAME_SPACE = multiprocessing.Manager().Namespace()
-PROCESSES: Dict[str, List[str]] = multiprocessing.Manager().dict()
+# Here we use MyPy ignore because multiprocessing.Manager().dict()
```

Comment:
Can we make it compatible? Do we need to cast it? What's the type of multiprocessing.Manager().dict()?